### ISSUE TITLE
isoline labels don't work in overlay

### ISSUE THREAD

**Issue by potter2** [UV-CDAT/uvcdat#1338]
I think I submitted this before but the isoline labels don't appear when an isoline plot is an overlay on a isofill  plot

**Comment by doutriaux1** [comment]
@remram44 assigning this to you guys because it works from the command line. It must have something to do with the order in which vistrails runs this.

![isolines_on_top](https://cloud.githubusercontent.com/assets/2781425/7738209/d64b1a82-ff06-11e4-8f8f-ea76f028300f.png)

**Comment by remram44** [comment]
Running the generated script from cdat, I don't see isolines either.
```python
import cdms2, cdutil, genutil, vcs

cdmsfile = cdms2.open('/home/remram/uvcdat-builds/build-2.2.0-gd124bd7/install/sample_data/clt.nc')
clt = cdmsfile('clt')
clt = clt(latitude=(-90.0, 90.0),squeeze=1,longitude=(-180.0, 175.0),time=('1979-1-1 0:0:0.0', '1988-12-1 0:0:0.0'),)

canvas = vcs.init()
gmIsoline = vcs.getisoline('default')
canvas.plot(clt, gmIsoline, cdmsfile=cdmsfile.id)
canvas.interact()
```

**Comment by doutriaux1** [comment]
well, yes, I don't know what this "cdmsfile=cdmsfile.id" is doing here. vcs probably error exit under the hood choking on this. So that's the issue.

**Comment by doutriaux1** [comment]
Just to be clear, the scritp/VisTrails should NOT add this argument. I don't even know what this is supposed to do... @ThomasMaxwell is it something for DV3D?

**Comment by doutriaux1** [comment]
@remram44 actually I'm not sure what you are talking about here, I do SEE isolines on the picture you posted...

**Comment by remram44** [comment]
Yes but I don't see any label.

**Comment by doutriaux1** [comment]
you need to turn them on manually. The default is no labels. I guess it's historical, users didn't want to see them on all the time.

**Comment by remram44** [comment]
So... what's the issue?

**Comment by doutriaux1** [comment]
@remram44 @potter2 I get it now... a bit slow... sorry... @potter2 are you saying that you did turn on the labels and they didn't show up?

**Comment by doutriaux1** [comment]
@potter2 I thought you meant if you plot isoline on top of isofill then you don't see the isolines.

**Comment by doutriaux1** [comment]
@potter2 ok I see your bug from command line now. re-assigning to myself @dlonie if you beat me to it, feel free. Here is a snippet of what goes wrong:
```
In [2]: iso = x.createisofill()

In [3]: iso2 = x.createisoline()

In [4]: iso2.label='y'

In [5]: x.plot(s,iso)
Out[5]: <vcs.displayplot.Dp at 0x11e15a9b0>

In [6]: x.plot(s,iso2)
Out[6]: <vcs.displayplot.Dp at 0x11e4ad140>

```
no labels (my guess is the priority is not set properly and they are bellow the isofills)

**Comment by potter2** [comment]
Yes

Sent from my iPhone

On May 21, 2015, at 11:42 AM, Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>> wrote:


@remram44<https://github.com/remram44> @potter2<https://github.com/potter2> I get it now... a bit slow... sorry... @potter2<https://github.com/potter2> are you saying that you did turn on the labels and they didn't show up?

**Comment by potter2** [comment]
No, the isolines show up but not the labels when they are turned on

Sent from my iPhone

On May 21, 2015, at 11:43 AM, Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>> wrote:


@potter2<https://github.com/potter2> I thought you meant if you plot isoline on top of isofill then you don't see the isolines.

**Comment by sankhesh** [comment]
Can this issue be closed now?

**Comment by potter2** [comment]
This still doesn’t work on the version I just built.

Jerry



Gerald (Jerry) Potter
NASA Goddard Space Flight Center
E-mail  gerald.potter@nasa.gov<mailto:gerald.potter@nasa.gov>
Office  28:S241
Phone    707.315.8314
Analysis and Data Consultant
Department      GSFC:606.2

On Aug 6, 2015, at 9:47 AM, Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #1338<https://github.com/UV-CDAT/uvcdat/issues/1338>.

**Comment by sankhesh** [comment]
@potter2 

What is the git SHA1-sum of your UV-CDAT source tree HEAD? 

If it is after 73617d1, then could you please post your test script for me to try?
